core/src/apps/tron/sign_tx.py

```
@@ -79,7 +79,7 @@ async def _require_confirm_by_type(ctx, transaction, owner_address):
 
         if action == "Transfer":
             token = tokens.token_by_address(
-                "TRC20", contract.trigger_smart_contract.contract_address
+                contract.trigger_smart_contract.contract_address
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)